src/git/zcl_abapgit_git_add_patch.clas.abap

```diff
@@ -49,7 +49,7 @@ CLASS ZCL_ABAPGIT_GIT_ADD_PATCH IMPLEMENTATION.
       CASE <ls_diff>-result.
         WHEN ' '.
 
-          INSERT <ls_diff>-new INTO TABLE rt_patch.
+          INSERT <ls_diff>-old INTO TABLE rt_patch.
```

Comment:
I'm a bit surprised that we didn't catch this before with the 14 UTs. Maybe add a testcase for this.

Reply:
Yeah, me too. UT added.